services/photoService.ts

```
@@ -40,3 +41,15 @@ export const getPhotos = async (albumId: string) => {
 		throw new Error(`予期せぬエラーが発生しました`);
 	}
 };
+
+export const addPhotos = createAsyncThunk(
+	"album/addPhotos",
+	async ({ photosList, albumId, uid }: AddPhotosRequest) => {
+		try {
+			await photoRepository.addPhotos({ photosList, albumId, uid });
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)